js/PageLevelProgressIndicatorView.js

```diff
@@ -0,0 +1,68 @@
+define([
+    'core/js/adapt'
+], function(Adapt) {
+
+    var PageLevelProgressIndicatorView = Backbone.View.extend({
+
+        initialize: function(options) {
+            options = options || {};
+            this.calculatePercentage = options.calculatePercentage || this.calculatePercentage;
+            this.ariaLabel = options.ariaLabel || "";
+            this.type = options.type || this.model.get("_type");
+            this.addClasses();
+            this.checkAria();
+            this.setUpEventListeners();
+            this.refresh();
+        },
+
+        addClasses: function() {
+            this.$el.addClass([
```

Comment:
Could we use `className`?

Reply:
Not for line 11 and line 21.
Box menu items would default to 'page' instead of menu if we couldn't override the type.